micromouse/RobotIO.h

```diff
@@ -87,7 +88,20 @@ namespace Micromouse
 		void followPath(Path * path);
 
 		void calibrateIRSensors();
-
+		
+		
+		
+		
+		/* Test hard ware calibration */
+		
+		bool checkMotors();
+		void checkIRSensors();
+		void checkGyro();
```

Comment:
need to all return bools, and should have a function called `checkSystem` that calls each one